src/tree_data.h

```diff
@@ -1440,6 +1440,28 @@ int lyd_lyb_data_length(const char *data);
  */
 void *lyd_set_private(const struct lyd_node *node, void *priv);
 
+/**
+ * @brief Get the data node based on a simple XPath.
+ * This API returns the closest parent of the node (or the node itself)
+ * identified by the nodeid (path). This API uses lyd_find_sibling() If cache is
+ * enabled and the siblings are NOT top-level nodes, this function finds the
+ * node in a constant time.
+ * @param[in] data_tree Existing data tree to get.
+ * @param[in] ctx Context to use.
+ * @param[in] path Simple data path (see @ref howtoxpath). List nodes can have predicates, one for each list key
+ * in the correct order and with its value as well or using specific instance position, leaves and leaf-lists
+ * can have predicates too that have preference over \p value. When specifying an identityref value in a predicate,
+ * you MUST use the module name as the value prefix!
+ * @param[in] value Value of the new leaf/lealf-list (const char*). If creating anydata or anyxml, the following
+ * \p value_type parameter is required to be specified correctly. If creating nodes of other types, the
+ * parameter is ignored.
+ * @param[in] value_type Type of the provided \p value parameter in case of creating anydata or anyxml node.
+ * @param[in] options Bitmask of options flags, see @ref pathoptions.
```

Comment:
The only relevant option is `LYD_PATH_OPT_OUTPUT` which you should specify explicitly. Even more, you could change the options to a single flag `int output` and then transform it internally into the flag.

Reply:
done